src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
```

Comment:
For future: don't do your home works in branch "master". Always create separate branch